File: src/cli/show_help.go
package cli

import (
	"fmt"
	"stylelintbc/src/constants"
)

func ShowHelp() {
	fmt.Println("Usage: stylelintbc [options]")
	fmt.Println()
	fmt.Println("Command line for easy Stylelint configuration")
	fmt.Println()
	fmt.Println("Options:")

	for _, option := range constants.Options {
		fmt.Printf("%-15s %-5s %s\n", option.Option, option.Alias, option.Description)
	}

	fmt.Println()
}

File: src/cli/show_help.rs
use crate::types;
use std::fmt::Write;

pub fn show_help(options: Vec<types::CliOption>) {
    let mut builder = String::new();

    builder.push_str("Usage: stylelintbc [options]\n\n");
    builder.push_str("Command line for easy Stylelint configuration\n\n");
    builder.push_str("Options:\n");

    for option in options {
        let _ = write!(
            builder,
            "{:<15} {:<5} {}\n",
            option.name, option.alias, option.description
        );
    }

    builder.push('\n');

    print!("{}", builder);
}
